src/main/java/Task.java

```diff
@@ -0,0 +1,54 @@
+public class Task {
+    protected String description;
+    protected String timing;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        addNewTask();
+    }
+
+    public Task(String description, String timing) {
+        this.description = description;
+        this.timing = timing;
+        this.isDone = false;
+        addNewTask();
+    }
```

Comment:
Calling addNewtTask() before initialisation of method leads to bad code structure. Might want to include it earlier.

Reply:
Thanks for pointing this out